R/test.r

```diff
@@ -68,6 +62,19 @@ test <- function(pkg = ".", filter = NULL, ...) {
     do.call(testthat::test_dir, testthat_args))
 }
 
+#' @export
+#' @rdname test
+test_coverage <- function(pkg = ".", ...) {
+  pkg <- as.package(pkg)
+
+  check_suggested("covr")
+
+  coverage <- covr::package_coverage(pkg$path, ...)
+  covr::shine(coverage)
```

Comment:
I would use `covr::report()`. I think I am going to be removing the `shine()` alias in the near future, as it no longer produces a shiny app, so doesn't really make sense.